auditbeat/auditbeat.reference.yml

```diff
@@ -29,60 +29,26 @@ auditbeat.max_start_delay: 10s
 #==========================  Modules configuration =============================
 auditbeat.modules:
 
-# The auditd module collects events from the audit framework in the Linux
```

Comment:
Did you have DEV_OS=darwin set when you updated? Looks like some thing that should be changed have been changed.

Reply:
It was `GOOS` and `GOARCH`, I think. I just resubmitted after generating without those, and it got rid of the config file weirdness.